lib/chef-acceptance/cli.rb

```diff
@@ -1,36 +1,17 @@
 require 'thor'
-require 'yaml'
+require 'chef-dk/chef_runner'
 
 class ChefAcceptance
   class Cli < Thor
+    ACCEPTANCE_COMMANDS = %w(destroy provision test)
 
-    attr_accessor :config
-
-    def initialize(*args)
-      super
-      root = ENV['ACCEPTANCE_ROOT'] || Dir.pwd
-      acceptance_yaml = File.join(root, '.acceptance.yml')
-      if !File.exist?(acceptance_yaml)
-        puts ".acceptance.yml file not found in #{root}. Make sure you are running from an acceptance directory."
-        exit 1
+    ACCEPTANCE_COMMANDS.each do |command|
+      desc command, "Run #{command}"
+      define_method(command) do |test_suite|
+        cookbook_path = File.join(test_suite, '.acceptance')
```

Comment:
I think we should check if `cookbook_path` exists here and fail with something along the lines of "Could not find #{test_suite} in the current working directory. Make sure to have a test suite with ".acceptance" directory."